Makefile.in

```diff
@@ -524,6 +524,7 @@ am__libmesh_dbg_la_SOURCES_DIST = src/base/default_coupling.C \
 	src/systems/explicit_system.C src/systems/fem_context.C \
```

Comment:
Makefile.in (and configure, and anything else generated by bootstrap) changes need to be in a separate commit from all other changes; and that commit needs to be after the Makefile.am changes that led to them, not before.